src/apiCalls.js

```diff
@@ -40,3 +40,38 @@ export const getPaletteById = (id) => {
   return fetch(process.env.REACT_APP_BACKEND_URL + `/api/v1/palettes/${id.id}`)
     .then(res => res.json());
 }
+
+export const deletePalette = (id, palette) => {
+  const options = {
+    method: 'DELETE',
+    body: JSON.stringify({
+      palette
+    }),
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  }
+
+  return fetch(process.env.REACT_APP_BACKEND_URL + `/api/v1/palettes/${id}`, options)
+    .then(res => res.json())
+}
+
+export const deleteProject = (id, project) => {
+  const options = {
+    method: 'DELETE',
+    body: JSON.stringify({
+      project
+    }),
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  }
+
+  return fetch(process.env.REACT_APP_BACKEND_URL + `/api/v1/projects/${id}`, options)
+    .then(res => res.json())
+}
+
+export const getPaletteByProjectId = (id) => {
```

Comment:
Nice work getting these into apiCalls file - we should try and test this file and our actions/reducers as a good next step.